src/3_branch.js

```diff
@@ -323,6 +338,36 @@ Branch.prototype['init'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
 }, true);
 
 /**
+ * @function Branch.addListener
+ * @param {function(String)} observer - _required_ - Observing function that will recieves a string as the events.
+ * @param {String} subject - _optional_ - Optionally specify which subjects you would like to observe. If left empty, the observer will recieve all events. Currently, the only Branch subject that publishes events is the banner.
+ *
+ */
+Branch.prototype['addListener'] = wrap(callback_params.NO_CALLBACK, function(done, observer, subject) {
```

Comment:
wrapping would put it in the queue - we want to keep it out of the queue, so callbacks get added immediately right?